### ISSUE TITLE
scam

### ISSUE THREAD

**Issue by Anonymusss12e** [LanifyAI/extension#1]
the take my lan token 10k lan and when i ask them they ban me on tg and discord , scamers

**Comment by CHEEMA2ZESTTY** [comment]
> the take my lan token 10k lan and when i ask them they ban me on tg and discord , scamers

is this true?

**Comment by Anonymusss12e** [comment]
Yes 2 time , 1st time they told me that they have a problem and to open a new acc and will give back my lan on the second acc, and second time after i ask when they will send my lan from 1st acc they take what i earn  and ban me on discord and tg
